examples/guestbook/README.md

```diff
@@ -67,7 +67,82 @@ The web front end interacts with the redis master via javascript redis API calls
 
 This example requires a running Kubernetes cluster.  See the [Getting Started guides](../../docs/getting-started-guides/) for how to get started. As noted above, if you have a Google Container Engine cluster set up, go [here](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead.
 
-### Step One: Start up the redis master
+### Step One: Fire up the redis master service
```

Comment:
In the Quick Start, we need to think about how to shortcut the discussion in "Accessing the guestbook site externally".

We could show how to access the guestbook through the apiserver and kubectl proxies, just to test/show that it works, and make a forward reference to the section with how to do it properly. If we use the proxies, we should also link to documentation about them: https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/accessing-the-cluster.md